config/http_config.go

```diff
@@ -74,6 +74,9 @@ type HTTPClientConfig struct {
 	ProxyURL URL `yaml:"proxy_url,omitempty"`
 	// TLSConfig to use to connect to the targets.
 	TLSConfig TLSConfig `yaml:"tls_config,omitempty"`
+	// DisableKeepAlives, if true, prevents re-use of TCP connections
+	// between different HTTP requests.
+	DisableKeepAlives bool
```

Comment:
Per previous (extensive) discussions, this shouldn't be a user-facing option. This should come from the code using it.

Reply:
Fine with me. So you wouldn't want this part of HTTPClientConfig but instead an additional arg to NewClientFromConfig?

Reply:
Yeah, that's one way to do it.